core/src/Kokkos_Core_fwd.hpp

```diff
@@ -121,9 +121,11 @@ class InitializationSettings;
   [[clang::annotate("DefaultExecutionSpace")]]
 #define KOKKOS_IMPL_DEFAULT_HOST_EXEC_SPACE_ANNOTATION \
   [[clang::annotate("DefaultHostExecutionSpace")]]
+#define KOKKOS_IMPL_SHARED_SPACE_ANNOTATION [[clang::annotate("SharedSpace")]]
```

Comment:
Why did you use a Clang annotation?

Reply:
symmetry with `DefaultExecSpace` etc.

Reply:
Drop it, the other one was helpful in the Kokkos specific ClangTidy checks Drew wrote in [kokkos/llvm-project](https://github.com/kokkos/llvm-project)